cli/pcluster/config/mappings.py

```diff
@@ -469,6 +471,24 @@
                 "allowed_values": ALLOWED_VALUES["per_unit_storage_throughput"],
                 "update_policy": UpdatePolicy.UNSUPPORTED
             }),
+            ("daily_automatic_backup_start_time", {
+                "allowed_values": r"NONE|^([01]\d|2[0-3]):([0-5]\d)$",
+                "update_policy": UpdatePolicy.UNSUPPORTED
```

Comment:
We decided to make it unsupported (as part of f4d38b36) because we couldn't guarantee that validation could be performed such that an update of this parameter wouldn't fail. The thought was that even though the config might be valid, the update might fail due to incompatibilities between the config and the existing CFN stacks.

However, I think I was wrong about this. Either config validation will fail, or other parameters for which update is unsupported (import_path, export_path, deplyoment_type, etc.) will prevent update. I'll change `copy_tags_to_backup`, `automatic_backup_retention_days`, and `daily_automatic_backup_start_time` to support update.

Reply:
Actually, update for `copy_tags_to_backup` will remain unsupported after looking at the CFN documentation.

Reply:
> However, I think I was wrong about this. Either config validation will fail, or other parameters for which update is unsupported (import_path, export_path, deplyoment_type, etc.) will prevent update. I'll change `copy_tags_to_backup`, `automatic_backup_retention_days`, and `daily_automatic_backup_start_time` to support update.

Yes, I think this is the right approach. Configuration consistency is left to validators; update policies should just tell if updating the parameter is supported, in principle.